app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):

```

Comment:
Add an ABC class after the player, so that another programmer knows, that your class is inheriting an abstract class!